egs/aishell/ASR/whisper/requirements.txt

```diff
@@ -0,0 +1,10 @@
+k2
+kaldialign
+git+https://github.com/lhotse-speech/lhotse
+sentencepiece
+tensorboard
+librosa
+git+https://github.com/yuekaizhang/whisper.git
```

Comment:
What changes have you made to the official `whisper`?

Reply:
https://github.com/openai/whisper/pull/1959 Here, otherwise, it can't support mutli-batch + multi-beam inference.